src/features/CommentSlice.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { getPostComments, createComment, deleteComment } from '../api/comments';
+// eslint-disable-next-line import/no-cycle
```

Comment:
better avoid circular imports
move the exported entity into a separate file and import into other files from there